test/e2e/test_sandbox.py

```
@@ -85,6 +97,24 @@ def test_sandbox_dryrun_custom_workdir(agent):
     assert "--workdir=/work" in result
 
 
+@pytest.mark.e2e
+@skip_if_no_container
+@pytest.mark.parametrize("agent", ["goose", "opencode"])
+def test_sandbox_dryrun_llm_endpoint(agent):
+    """--llm-endpoint should overwrite the openai endpoint url."""
+    result = check_output(_dryrun_cmd(agent) + ["--llm-endpoint", "http://model.server.local:8321/"])
```

Ruff (0.15.18)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)